soh/soh/Enhancements/randomizer/randomizer.cpp

```diff
@@ -3603,6 +3613,9 @@ void GenerateRandomizerImgui() {
     cvarSettings[RSK_KAK_GATE] = CVar_GetS32("gRandomizeKakarikoGate", 0);
     cvarSettings[RSK_DOOR_OF_TIME] = CVar_GetS32("gRandomizeDoorOfTime", 0);
     cvarSettings[RSK_ZORAS_FOUNTAIN] = CVar_GetS32("gRandomizeZorasFountain", 0);
+    //Starting Age is forced to child if forest setting is set to closed. (0 = Adult, 1 = Child)
+    cvarSettings[RSK_STARTING_AGE] = ((CVar_GetS32("gRandomizeForest", 0) != 0) && 
+                                        (CVar_GetS32("gRandomizeStartingAge", 1)));
```

Comment:
No need for the != 0 in here.

Also, gRandomizeStartingAge should be 0 at default if you want to swap the ages around in the options menu (see below).